Sources/SpeziOnboarding/OnboardingConstraint.swift

```diff
@@ -23,15 +23,7 @@ import Spezi
 public protocol OnboardingConstraint: Standard {
     /// Adds a new exported consent form represented as `PDFDocument` to the `Standard` conforming to ``OnboardingConstraint``.
     /// 
-    /// - Parameters:
-    ///     - consent: The exported consent form represented as `PDFDocument` that should be added.
-    @available(
-        *,
-        deprecated,
-        message: """
-        Storing consent documents without an identifier is deprecated.
-        Please use `ConsentConstraint.store(consent: PDFDocument, identifier: String)` instead.
-        """
-    )
+    /// - Parameter consent: The exported consent form represented as `PDFDocument` that should be added.
+    @MainActor
```

Comment:
I think we have lost some of the deprecation warnings and documentation in the merge:
```suggestion
    /// - Parameters:
    ///     - consent: The exported consent form represented as `PDFDocument` that should be added.
    @available(
        *,
        deprecated,
        message: """
        Storing consent documents without an identifier is deprecated.
        Please use `ConsentConstraint.store(consent: PDFDocument, identifier: String)` instead.
        """
    )
    @MainActor
```

Reply:
Whoops, sorry about that!

Reply:
No worries; happens in merge commits all the time 👍